src/partials/messenger.navigation.html

```diff
@@ -64,7 +64,7 @@
         </md-button>
     </div>
     <div class="search" searchbox searchbox-focus="ctrl.searchVisible">
-        <input type="text" ng-model="ctrl.searchText" translate translate-attr-placeholder="messenger.SEARCH" aria-label="Search" translate-attr-aria-label="messenger.SEARCH">
+        <input type="search" ng-model="ctrl.searchText" translate translate-attr-placeholder="messenger.SEARCH" aria-label="Search" translate-attr-aria-label="messenger.SEARCH" ng-keyup="ctrl.clearSearch($event)">
```

Comment:
There's one last remaining problem: With the change in https://github.com/threema-ch/threema-web/pull/1026/commits/837762a4b2efa403787de6f32bc58d5dcc5acdb6, pressing ESC in Chrome with a filled search box will immediately close it.

I believe this is because search fields are automatically cleared by Chrome when pressing ESC, so at the time of the keyup event, it's already empty.

The solution to this is to change the line above from ng-keyup to ng-keydown. This will also result in a _slightly_ faster reaction.